src/components/MerchantsPage/MerchantsPage.tsx

```diff
@@ -95,8 +107,11 @@ const MerchantsPage = (props: Props) => {
               {/* TODO: hook this part up to actual amounts - is there a total amount api call? */}
```

Comment:
I think you can remoce this TODO now if you want ;)

Reply:
done